docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+If you like to make changes to the source code, please follow these contribution guidelines.
+
+## How to contribute?
+
+<!-- markdown-link-check-disable -->
+Follow the [CGDS' best practices for code
+development](http://cgds.uab.edu/manual-of-operations/standards-definitions/#standard-practices) (see sections IDE,
```

Comment:
I don't think this link is accessible publicly, likely being flagged as dead consider removing.

Reply:
Love the idea. I have now modified `contributing.md` based on rosalution's. Also added template for `issues` based on snakemake repo.